tests/lib/rules/no-extra-parens.js

```diff
@@ -120,6 +121,8 @@ ruleTester.run("no-extra-parens", rule, {
         "a(b)(c)",
         "a((b, c))",
         "new new A",
+        { code: "1 ** 2 ** 3", parserOptions: { ecmaVersion: 7 } },
+        { code: "1 ** (2 ** 3)", parserOptions: { ecmaVersion: 7 } },
```

Comment:
This should be invalid -- the parens are unnecessary in that case because `**` is right-associative.

```js
2 ** 3 ** 4

// is equivalent to

2 ** (3 ** 4)
```

So the existing rule also behaves incorrectly in this case:

```js
(2 ** 3) ** 4 // this should be valid
```